src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.NamespaceWithWorkflow
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
```

Comment:
Just for clarification... this will be a workaround until generic model implementation take place in cromwell-engine.